tripy/tests/integration/test_functional.py

```diff
@@ -78,59 +78,6 @@ def test_multi_output_trace(self):
             and (cp.from_dlpack(out[1]) == cp.array([4.0, 4.0], dtype=np.float32)).all()
         )
 
-    def _test_framework_interoperability(self, data, device):
-        a = tp.Tensor(data, device=device)
-        if device.kind == "gpu":
-            b = tp.Tensor(torch.tensor(data).to("cuda"), device=device)
-        else:
-            b = tp.Tensor(torch.tensor(data), device=device)
-
-        out = a + b
-        assert (cp.from_dlpack(out).get() == np.array([2.0, 2.0], dtype=np.float32)).all()
-
-    def test_cpu_and_gpu_framework_interoperability(self):
-        self._test_framework_interoperability(np.ones(2, np.float32), device=tp.device("cpu"))
-        self._test_framework_interoperability(cp.ones(2, cp.float32), device=tp.device("gpu"))
-
-    def _test_round_tripping(self, data, device):
-        assert isinstance(data, np.ndarray) or isinstance(data, cp.ndarray)
-
-        # Assert round-tripping for numpy or cupy array
-        xp_orig = data
-        if device.kind == "gpu":
-            xp_round_tripped = cp.array(cp.from_dlpack(tp.Tensor(xp_orig, device=device)))
-        else:
-            xp_round_tripped = np.array(np.from_dlpack(tp.Tensor(xp_orig, device=device)))
-        assert (xp_round_tripped == xp_orig).all()
-        # (39): Remove explicit CPU to GPU copies. Add memory pointer checks.
-        # assert xp_round_tripped.data == xp_orig.data
-
-        # Assert round-tripping for Torch tensor
-        torch_orig = torch.as_tensor(data)
-        if device.kind == "gpu":
-            torch_round_tripped = torch.as_tensor(cp.from_dlpack(tp.Tensor(torch_orig.to("cuda"), device=device)))
-        else:
-            torch_round_tripped = torch.as_tensor(np.from_dlpack(tp.Tensor(torch_orig, device=device)))
-        assert torch.equal(torch_round_tripped, torch_orig)
-        # (39): Remove explicit CPU to GPU copies. Add memory pointer checks.
-        # Below fails as we do allocate a new np array from Torch tensor data.
-        # assert torch_data_round_tripped.data_ptr == torch_data.data_ptr
-
-        # (39): Remove explicit CPU to GPU copies. Add memory pointer checks.
-        # Figure out how to compare two Jax data memory pointers.
-
-        # Assert round-tripping for List data
-        if device.kind == "cpu":
-            list_orig = data.tolist()
-            list_round_tripped = np.from_dlpack(tp.Tensor(list_orig, shape=(2,), device=device)).tolist()
-            assert list_round_tripped == list_orig
-            # (39): Remove explicit CPU to GPU copies. Add memory pointer checks.
-            # assert id(list_round_tripped) == id(list_orig)
-
-    def test_tensor_round_tripping(self):
```

Comment:
Why are we removing this?

Reply:
Already have coverage in `tripy/tests/frontend/test_tensor.py`.